### PR TITLE
Session deep links

### PR DESCRIPTION
**Pull request by gewfy**
- [x] Dynamic links util
- [x] Create link on session creation
- [x] Update docs about `.env` changes
- [x] Update GitHub secrets
- [x] Update workflows with new `.env` variables

### CODE REVIEW

File: functions/src/api/sessions/index.ts [hunk 1]
```diff
@@ -84,12 +85,15 @@ sessionsRouter.post('/', validator({body: CreateSessionSchema}), async ctx => {
     timestamp: Timestamp.now(),
   };
 
+  const link = await createDynamicLink(`sessions/${data.id}`);
```
Comment (Nipher) [inline functions/src/api/sessions/index.ts:88]: I imagine that would be the invitecode instead of the id for private sessions in the future? 🤔

**Review by Nipher — APPROVED** [review]
super cool!

File: functions/src/api/sessions/index.ts (hunk 1 above)
Comment (Nipher) [reply functions/src/api/sessions/index.ts:88]: actually both would work since we could fetch the data from either 🙌

Comment (gewfy) [reply functions/src/api/sessions/index.ts:88]: Yeah I was thinking the same. That for private sessions we could have another pattern. E.g. `sessions/:sessionId/invitation/:invitationCode` or similar.

Comment (gewfy) [reply functions/src/api/sessions/index.ts:88]: This is where you later setup the routing in the client:

https://github.com/29ki/29k/blob/08892f96c351a0e71dc1854c441d2b1c65f4b977/client/src/lib/navigation/linking.ts#L10-L19

**Pull request merged**